.github/workflows/test-codeql.yml

```
@@ -0,0 +1,64 @@
+name: "Test CodeQL Configuration"
+
+on:
+  workflow_dispatch:  # Manual trigger only
+  push:
+    branches: [ "main", "develop" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)